src/Augurk.CommandLine/Commands/PublishCommand.cs

```diff
@@ -113,6 +113,11 @@ private int PublishFeatureFiles()
                             return -1;
                         }
                     }
+                    catch (InvalidOperationException)
+                    {
+                        Console.Error.WriteLine($"Unable to parse feature file '{featureFile}'. Possible causes are that the feature file is commented out entirely.");
+                        return -1;
```

Comment:
Return when a single file has no valid content?
Shouldn't we at least provide the option to skip such files?

Reply:
Skipping them for now, but writing a warning, which should be picked up by the build pipeline (hopefully)